index.html

```diff
@@ -0,0 +1,205 @@
+                    <button>See Project</button>
+                </div>
+                <div class=" work wdr23">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class="work wdr31">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+
+                <div class="work wdr32">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class="work wdr33">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                </div>
+
+        </div>
+
+    </div>
+    <!------------------about me------------------------------------------------>
+    <div class="aboutme-container">
+                <div class="aboutme-left">
+                <h4>About me</h4>
+                <p>Hey there! I can help you build a product, feature or website. Look through some of my work and
+                    experience! If you like
+                    what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="aboutme-right">
+                <button class="gr">Get My Resume</button>
+            </div>
+
+        <div class="hline">
+            <hr>
+        </div>
```

Comment:
- [ ] Kindly adjust this section of your code to make your work look really great and neat by strictly following the Figma design :point_down: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/213870455-8b17d5b2-2f37-4864-8140-e5b97623dde2.png)|![image](https://user-images.githubusercontent.com/99715640/213870531-095e9a6a-74b8-401d-aecf-de1d398c6b79.png)|